grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ReactorGrpcClientStubFactory.java

```diff
@@ -45,7 +45,14 @@ public ServiceDescriptor findServiceDescriptor(Class<?> clientType) {
             return (ServiceDescriptor) getServiceDescriptorMethod.invoke(null);
         } catch (NoSuchMethodException | IllegalAccessException |
                 InvocationTargetException | NoSuchFieldException e) {
-            return null;
+            Throwable t = e;
+            if (e instanceof InvocationTargetException) {
```

Comment:
nit; What do you think of using `Exceptions.peel` instead? Ditto for the other occurrences

Reply:
I forgot about the API. 😓